### DISCUSSION TITLE
Tuya smart WIFI IR control

### DISCUSSION THREAD

**Discussion by bartplessers** [arendst/Tasmota#14225] [Q&A] [answered]
Hi,

Does anybody have some experience with flashing

![image](https://user-images.githubusercontent.com/10470708/147847713-58b18fab-68aa-4f6c-9774-7d828cc76ebc.jpeg)

https://m.banggood.com/Tuya-Smart-WIFI-IR-Remote-Control-Universal-Infrared-Wireless-Remote-Controller-for-TV-DVD-AUD-AC-Work-with-Alexa-Google-Home-via-Smart-Life-Tuya-APP-p-1830494.html

It looks like it can be flashed with [OpenMqttGateway](https://1technophile.blogspot.com/2020/07/avatto-s06-ir-gateway-compatible-with.html?m=1), but all my other wifi-devices are running on Tasmota and I feel very comfortable with that, so I would love to have the same firmware on this IR-blaster.

Anybody experience with this device?

Kind regards 
Bart

**Comment by barbudor** [comment, marked as answer]
Did you searched in the template repository?
https://templates.blakadder.com/misc.html

Anyhow, looking at the OMG wiki you linked, it seems perfectly compatible with just the ESP and the GPIOs are listed.

**Comment by Jason2866** [comment]
The IR receiver use the GPIO5,
the IR emitters is on GPIO14, 
the blue LED on GPIO4
the reset button on GPIO13.
